views/templates/hook/ps_googleanalytics.tpl

```diff
@@ -17,39 +17,57 @@
  * International Registered Trademark & Property of PrestaShop SA
  *}
 
+{if $isV4Enabled}
+  <script async src="https://www.googletagmanager.com/gtag/js?id={$gaAccountId|escape:'htmlall':'UTF-8'}"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    {literal}function gtag(){dataLayer.push(arguments);}{/literal}
+    gtag('js', new Date());
+    gtag(
+      'config',
+      '{$gaAccountId|escape:'htmlall':'UTF-8'}',
+      {ldelim}
+        'debug_mode':true
```

Comment:
debug mode should not be hardcoded to true. It can be activated with a chrome extension.